client/verta/tests/modelapi_hypothesis/value_generator.py

```diff
@@ -43,9 +43,9 @@ def value_from_api(api):
 
 def verta_type_to_dtype(name):
     map = {
-        "VertaBool": np.bool,
-        "VertaFloat": np.float,
-        "VertaString": str,
+        "VertaBool": np.bool_,
+        "VertaFloat": np.float_,
+        "VertaString": np.str_,
```

Comment:
This fixes

```
modelapi_hypothesis/test_value_generator.py::test_series_from_api
modelapi_hypothesis/test_modelapi.py::test_dataframe_modelapi_and_values
modelapi_hypothesis/test_value_generator.py::test_dataframe_from_api
```

which threw

```
AttributeError: module 'numpy' has no attribute 'bool'.
`np.bool` was a deprecated alias for the builtin `bool`. To avoid this error in existing code, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
```